src/Control/Abstract/Modules.hs

```diff
@@ -31,7 +31,7 @@ import Source.Span
 import System.FilePath.Posix (takeDirectory)
 
 -- A scope address, frame address, and value ref.
-type ModuleResult address value = ((address, address), value)
```

Comment:
I was trying to `fmap` over analysis results to erase the term type, which didn’t end up working out. The attempt was made more convenient by eta-reducing this, so I’ve left it thus.

Reply:
Just tried; nope.

I’ll add a comment.